lib/src/webcrypto/webcrypto.hmac.dart

```diff
@@ -197,7 +197,7 @@ abstract class HmacSecretKey {
   /// timing attacks. To validate signatures use [verifyBytes] or [verifyStream]
   /// instead, these methods computes a signature and does a
   /// fixed-time comparison.
-  /// {@template}
+  /// {@endtemplate}
```

Comment:
For future reference please submit unrelated changes as individual PRs.

That makes it much easier to review and land them, and reduces the risk of conflicts, etc.

Small and fast is always preferable.